_includes/v20.2/misc/tooling.md

```diff
@@ -11,27 +11,23 @@ If you encounter problems, please [open an issue](https://github.com/cockroachdb
+| JavaScript/TypeScript | [Sequelize](https://www.npmjs.com/package/sequelize)<br><br><hr>[TypeORM](https://www.npmjs.com/package/typeorm) | {% remote_include https://raw.githubusercontent.com/cockroachdb/cockroach/master/pkg/cmd/roachtest/sequelize.go |var supportedSequelizeRelease = "|"\n\n %} or later <br>[sequelize-cockroachdb](https://www.npmjs.com/package/sequelize-cockroachdb) (latest)<hr> {% remote_include https://raw.githubusercontent.com/cockroachdb/cockroach/master/pkg/cmd/roachtest/typeorm.go |var supportedTypeORMRelease = "|"\n\n %} or later  | Beta<br><br><hr>Full |
+| Ruby | [ActiveRecord](https://rubygems.org/gems/activerecord)<br>[RGeo/RGeo-ActiveRecord](https://github.com/cockroachdb/activerecord-cockroachdb-adapter#working-with-spatial-data) | [activerecord 5.2](https://rubygems.org/gems/activerecord/versions) or later<br>[activerecord-cockroachdb-adpater 5.2.2](https://rubygems.org/gems/activerecord-cockroachdb-adapter/versions) or later <hr>activerecord {% remote_include https://raw.githubusercontent.com/cockroachdb/cockroach/master/pkg/cmd/roachtest/activerecord.go |var supportedRailsVersion = "|"\nvar %} or later <br>activerecord-cockroachdb-adapter {% remote_include https://raw.githubusercontent.com/cockroachdb/cockroach/master/pkg/cmd/roachtest/activerecord.go |var activerecordAdapterVersion = "|"\n\n %} or later  | Full<br><br><hr>Full |
+| Python | [Django](https://pypi.org/project/Django/)<br>(including [GeoDjango](https://docs.djangoproject.com/en/3.1/ref/contrib/gis/))<hr>[peewee](http://docs.peewee-orm.com/en/latest/peewee/playhouse.html#cockroach-database)<hr>[PonyORM](build-a-python-app-with-cockroachdb-pony.html)<hr>[SQLAlchemy](build-a-python-app-with-cockroachdb-sqlalchemy.html) | Django {% remote_include https://raw.githubusercontent.com/cockroachdb/cockroach/master/pkg/cmd/roachtest/django.go |var djangoCockroachDBSupportedTag = "|"\n\n %} or later <br>[django-cockroachdb](https://pypi.org/project/django-cockroachdb/) {% remote_include https://raw.githubusercontent.com/cockroachdb/cockroach/master/pkg/cmd/roachtest/django.go |var djangoSupportedTag = "cockroach-|"\nvar %}<hr>[3.13.3](https://pypi.org/project/peewee/)<hr>[0.7.13](https://pypi.org/project/pony/)<hr>[SQLAlchemy 1.3.17](https://pypi.org/project/SQLAlchemy/)<br>[sqlalchemy-cockroachdb 1.3.0](https://pypi.org/project/sqlalchemy-cockroachdb/) | Full<br><br> <hr>Full<hr>Full<hr>Full |
```

Comment:
> var djangoSupportedTag = "cockroach-3.2.x"
> var djangoCockroachDBSupportedTag = "3.2.1"

is correct -- not super obvious but `cockroach-3.2.x` is just a branch of django that has testing changes to work with cockroachdb. `3.2.1` is the version of our adapter.

actually, we should always be recommending the latest versions of all our adapters (sqlalchemy-cockroachdb, django-cockroachdb, sequelize-cockroachdb, activerecord-cockroachdb-adapter, hibernate cockroach dialect). it would be more important to make sure the ORM version we list here is fixed.

maybe for those it's easier to just link to the github/pip/rubygem/npmjs pages?